src/components/Layout/Drawer.jsx

```diff
@@ -104,5 +101,9 @@ const ListText = styled(ListItemText)`
   font-weight: 700;
   font-size: 20px;
 `;
-
+const DrawerSlideBox = styled(DrawerSlide)`
```

Comment:
```suggestion
const DrawerWrapper = styled(Drawer)`
```
컴포넌트 네이밍도 일관성있게 하도록 중석이랑 컨벤션을 최대한 정해가면서 해야할듯

가급적 해당 파일을 MenuDrawer.jsx , 컴포넌트 이름도 MenuDrawer로 변경하고 
`import Drawer from "@mui/material/Drawer";` 처럼 그래도 import 해서 사용하도록 변경